tools/snippets/test/fixtures/julia/runner.jl

```diff
@@ -19,7 +19,7 @@
 import JSON
 
 """
-    gen( x, name )
+	gen( x, name )
```

Comment:
let's revert this change unrelated to the addressed issue.

```suggestion
    gen( x, name )
```